packages/core/src/components/event-list-item/event-list-item.tsx

```diff
@@ -37,7 +37,7 @@ export class EventListItem {
    * You can find a list of all available colors in our documentation.
    * Example values are `--theme-color-alarm` or `color-alarm`
    *
-   * {@link https://ix.siemens.io/docs/theming/colors/}
+   * {@link https://ix.siemens.io/docs/styles/colors#status-colors}
```

Comment:
No need to link to status colors since custom colors are also allowed
```suggestion
   * {@link https://ix.siemens.io/docs/styles/colors}
```